static/app/views/issueDetails/groupActivityItem.tsx

```diff
@@ -391,6 +392,17 @@ function GroupActivityItem({activity, organization, projectId, author}: Props) {
               author,
             });
       }
+      case GroupActivityType.SET_ESCALATING: {
+        return tct(
+          '[author] flagged this issue as escalating because over [forecast] [event] happened in an hour',
+          {
+            author,
+            forecast: activity.data.forecast,
```

Comment:
only activities associated with groups that were `archived until escalating` -> `escalating` have a `forecast`.  I think you'd need to check for it like what's done [here](https://github.com/getsentry/sentry/pull/49842/files#diff-fb445cb3dfcfb42e8203cbcc1ab9900911ffe8c9545b3631e7e617c72cc23afbL272) and return the [default](https://github.com/getsentry/sentry/pull/49842/files#diff-fb445cb3dfcfb42e8203cbcc1ab9900911ffe8c9545b3631e7e617c72cc23afbL282) if no forecast is found